src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/TransferFunded.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@model SFA.DAS.ProviderApprenticeshipsService.Web.Models.TransferFundedViewModel
+
+    @{
+        ViewBag.Title = "Transfer-funded cohorts";
+        ViewBag.PageId = "transfer-funded-cohorts";
+    }
+
+<div class="grid-row">
+    <div class="column-full">
+        
+        <h1 class="heading-xlarge">Transfer-funded cohorts</h1>
+
+        @{ var commitmentsCount = Model.Commitments.Count(); }
+        <p class="dynamic-heading">You have <strong>@commitmentsCount</strong> cohort@(commitmentsCount > 1?"s":"") to be paid via a transfer</p>
+
+        
+        <table class="tableResponsive employerListing">
+            <thead>
+            <tr>
+                <th>Employer</th>
+                <th>Reference</th>
+                <th>Transfer Status</th>
+                <th></th>
+
+            </tr>
+            </thead>
+            <tbody>
+            @foreach (var commitment in Model.Commitments)
+            {
+                <tr>
+                    <td>
+                        @commitment.ReceivingEmployerName &nbsp;
+                    </td>
+                    <td>
+                        @commitment.HashedCommitmentId &nbsp;
+                    </td>
+                    <td>
+                        @commitment.Status &nbsp;
+                    </td>
+                    <td>
+                        <a href="@Url.Action("Details", new {Model.ProviderId, commitment.HashedCommitmentId})" title="View  @commitment.ReceivingEmployerName" aria-label="View  @commitment.ReceivingEmployerName">Details</a>
```

Comment:
looks like 2 spaces between View and @commitment... in the title.

Reply:
... and the aria-label

Reply:
done